README.md

```diff
@@ -45,11 +45,11 @@ console.log(isoxmlManager.getWarnings())
 console.log(isoxmlManager.options)
 
 // get all the Partfileds
-const partfields: Partfield[] = isoxmlManager.rootElement.attributes.Partfield || []
+var partfields: Partfield[] = isoxmlManager.rootElement.attributes.Partfield || []
```

Comment:
@obermeier @Frank-Wiebeler Hmm, why do you propose to use `var` here instead of `const`?